browser/src/control/Parts.js

```diff
@@ -487,9 +487,26 @@ L.Map.include({
 				}
 			};
 
-			this.uiManager.showInfoModal('show-sheets-modal', '', ' ', ' ', _('Close'), callback, true, 'show-sheets-modal-response');
+			this.uiManager.showInfoModal('show-sheets-modal', 'Show sheets', ' ', ' ', _('OK'), callback, true, 'show-sheets-modal-response');
```

Comment:
_('Show sheets')
to enable localization

Reply:
Updated to enable localization, thanks @timar :)